Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -254,6 +254,8 @@ if(CGAL_Qt5_FOUND AND Qt5_FOUND AND OPENGL_FOUND AND QGLVIEWER_FOUND)
   target_link_libraries(scene_polyhedron_selection_item scene_polyhedron_item_decorator scene_polyhedron_item_k_ring_selection)
 
   add_item(scene_polyhedron_shortest_path_item Plugins/Surface_mesh/Scene_polyhedron_shortest_path_item.cpp)
+
+  add_item(scene_surface_mesh_item Scene_surface_mesh_item.cpp)
   
```

Comment:
You are right. I mixed it up with the line before:
```cmake
add_item(scene_polyhedron_shortest_path_item Plugins/Surface_mesh/Scene_polyhedron_shortest_path_item.cpp)
```
that does not depend on this PR.